test/cli/install/bun-remove.test.ts

```diff
@@ -191,7 +191,7 @@ it("should not reject missing package", async () => {
     stderr: "pipe",
     env,
   });
-  expect(await rmExited).toBe(0);
+  expect(await rmExited).toBe(1);
```

Comment:
Does `npm`/`pnpm`/`yarn` error as well? Why did this test exist in the first place

Reply:
`pnpm`/`yarn` do, `npm` does not
if users want the `npm` behavior they can do `bun remove <pkg> || true`